hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/src/test/java/org/apache/hadoop/yarn/applications/unmanagedamlauncher/TestUnmanagedAMLauncher.java

```diff
@@ -149,24 +151,24 @@ public void launchAM(ApplicationAttemptId attemptId)
               throws IOException, YarnException {
             YarnApplicationAttemptState attemptState =
                 rmClient.getApplicationAttemptReport(attemptId)
-                  .getYarnApplicationAttemptState();
-            Assert.assertTrue(attemptState
-              .equals(YarnApplicationAttemptState.LAUNCHED));
+                    .getYarnApplicationAttemptState();
+            assertEquals(attemptState, YarnApplicationAttemptState.LAUNCHED);
```

Comment:
Could you reverse the order of the arguments? The order is expected, actual.

Reply:
addressed